ffi-build.sh

```diff
@@ -46,6 +48,10 @@ sed < ddprof_ffi.pc.in "s/@DDProf_FFI_VERSION@/${version}/g" \
     | sed "s/@DDProf_FFI_LIBRARIES@/${native_static_libs}/g" \
     > "$destdir/lib/pkgconfig/ddprof_ffi.pc"
```

Comment:
The concept of "native static libs" doesn't apply to a shared object. There will be dependencies, of course, but this isn't how we should be populating them for the `.so`.

Reply:
Thanks for pointing this out. I did put them in `Libs.private` when I had still hope to be able to make a single pkg-config file for both shared and static library, but since I could not do it, it's better to completely remove them for the shared library.